src/dynamics/_common/track_tags.js

```diff
@@ -405,30 +451,30 @@ Scoped.define("module:TrackTags", [
              * @private
              */
             _loadMetaTrackTags: function() {
-                if (this._video)
+                if (this._video) {
                     Objs.iter(this._video.textTracks, function(track, index) {
                         if (typeof this._video.textTracks[index] === 'object' && this._video.textTracks[index]) {
-                            var _track = this._video.textTracks[index];
+                            const _track = this._video.textTracks[index];
                             // If set custom style to true show cue text in our element
                             if (_track.kind === 'metadata') _track.mode = 'hidden';
                             if (_track.kind === 'chapters') _track.mode = 'showing';
                         }
                     }, this);
+                }
             },
 
             /**
              * Will set default language text track
              * @private
              */
             _setDefaultTrackOnPlay: function() {
```

Comment:
@sambua since this isn't related to play action anymore I'd rename the function to something else
```suggestion
            _displayCustomTracks: function() {
```

Reply:
Had to replace where it's calling so will commit under other commit.